Lib9c/Policy/BlockPolicySource.cs

```diff
@@ -44,6 +44,8 @@ public partial class BlockPolicySource
 
         public const int MaxTransactionsPerBlock = 100;
 
+        public const long PreviewNetAdminAccountAllowIndex = 823_000;
```

Comment:
Since the `previewnet` chain index is greater than this value, does this mean that the `CreatePreviewNetAdminAccount` action will be obsolete because the admin account is already created?

Reply:
yes... It's already been a while, need to modify the index